apps/api_web/lib/api_web/controllers/prediction_controller.ex

```diff
@@ -61,6 +61,12 @@ defmodule ApiWeb.PredictionController do
     filter_param(:id, name: :route)
     filter_param(:id, name: :trip)
 
+    parameter("filter[route_pattern]", :query, :string, """
+    Filter by `route_pattern_id`: returns predictions associated with a trip with the given `route_pattern_id`. Multiple `route_pattern_id` #{
```

Comment:
Probably should say something like "Filter by `/included/{index}/relationships/route_pattern/data/id`" to be consistent with description for other filters, there is no `route_pattern_id` field in response. 

<img width="1389" alt="image" src="https://user-images.githubusercontent.com/45011335/61469689-a4903180-a94d-11e9-8884-a9482f978f4c.png">

Reply:
I wasn't sure what to do here since `route_pattern_id` isn't in attributes or in relationships. (It's the associated trip's `route_pattern_id`.)

Does it make sense to say something like: Filter by `/included/{index}/relationships/trip/included/{index}/route_pattern/id`?

Reply:
I think the path here should correspond with actual path in the result, which would be `/included/{index}/relationships/route_pattern/data/id`

<img width="979" alt="image" src="https://user-images.githubusercontent.com/45011335/61473629-c9d46e00-a954-11e9-8b53-e3962c17a9a2.png">